odr-audioenc.spec

```diff
@@ -93,7 +93,7 @@ file, a pipe, or to a ZeroMQ output compatible with ODR-DabMux.
 
 %package -n     %{toolame_dab_name}
 Version:        %{libtoolame_dab_version}
-Release:        %{libtoolame_dab_release}
+Release:        %{libtoolame_dab_release}.1
```

Comment:
Same here, can we add this to `%define libtoolame_dab_release`?

Reply:
Same as before - thanks.